src/components/Form/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+
+import Logo from "../../assets/bridge-logo.png";
+
+const Header = () => {
+  return (
+    <Container>
+      <div className="wrapper">
+        <h1>
+          <img src={Logo} alt="placeholder bridge logo" /> BridgeBot{" "}
```

Comment:
alt text should simply describe the image; in this case you could just write `"bridgebot logo"`